src/import/FSHImporter.ts

```diff
@@ -792,25 +793,9 @@ export class FSHImporter extends FSHVisitor {
   }
 
   private parseCodeLexeme(conceptText: string, parentCtx: ParserRuleContext): FshCode {
-    const splitPoint = conceptText.match(/(^|[^\\])(\\\\)*#/);
-    let system: string, code: string;
-    if (splitPoint == null) {
-      system = '';
-      code = conceptText.slice(1);
-    } else {
-      system = conceptText.slice(0, splitPoint.index) + splitPoint[0].slice(0, -1);
-      code = conceptText.slice(splitPoint.index + splitPoint[0].length);
-    }
-    system = system.replace(/\\\\/g, '\\').replace(/\\#/g, '#');
-    if (code.startsWith('"')) {
-      code = code
-        .slice(1, code.length - 1)
-        .replace(/\\\\/g, '\\')
-        .replace(/\\"/g, '"');
-    }
-    const concept = new FshCode(code);
-    if (system.length > 0) {
-      concept.system = this.aliasAwareValue(parentCtx, system);
+    const concept = parseCodeLexeme(conceptText);
```

Comment:
I refactored the core `parseCodeLexeme` logic into a separate shareable function since I needed it in the config importer too.